src/Informer/CMakeLists.txt

```diff
@@ -3,20 +3,42 @@
 
 set(LIBRARY Informer)
 
-set(LIBRARY_SOURCES
+add_spectre_library(${LIBRARY})
+
+# Note that this library has two special source files: InfoAtCompile.cpp and
+# InfoAtLink.cpp. These files contain placeholder strings that CMake fills in
+# (in copies of the files in the build directory) when configuring the build.
+#
+# We track the *generated* IntoAtCompile.cpp as a library source. This means
+# that CMake must have been configured *before* checking library dependencies
+# via the CMake function check_spectre_libs_dependencies.
+#
+# We do *not* track InfoAtLink.cpp as a library source, as it is used only to
+# inject information late in the build process. But note that InfoAtLink.cpp
+# depends on Boost.
```

Comment:
@nilsdeppe Here's the new comment. Do you feel this is an accurate (and helpful?) summary of the situation? If you  have any suggestions to improve or clarify, I'm happy to implement them 🙂

Reply:
Looks good to me!